src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/analyze_confomance_fails.py

```diff
@@ -0,0 +1,237 @@
+OLD_API_REG_EXP = '(ie_plugin/|ie_executable_network/|ie_infer_request/|conformance_query_model)'
+
+HW_DEVICES = ['TEMPLATE', 'CPU', 'VPUX', 'GNA', 'GPU']
+
+
+def set_log_local_dir(user_log_dir, remote_log_path, job_list):
+    log_dir = os.path.join(os.getcwd())
+    if user_log_dir:
+        if os.path.exists(user_log_dir):
+            log_dir = user_log_dir
+        else:
+            print(f'Local log dir {user_log_dir} is not exists')
+            exit(1)
+
+    if remote_log_path and job_list:
+        log_dir = os.path.join(log_dir, NEW_LOG_DIR)
+
+    return log_dir
+
+
+def process_remote_log(remote_log_path, job_list, local_log_dir):
+    if remote_log_path and job_list:
+        if 'linux' not in remote_log_path:
+            print(f'Sorry, I can analyze just linux results now')
+            exit(1)
+
+        for job in job_list:
+            job_number = ''.join(job)
+            job_name = f'{JOB_PREFIX_LINUX}{job_number}'
+            source_path = os.path.join(remote_log_path, job_name, LOGS_ZIP_NAME)
+            dest_path = os.path.join(local_log_dir, job_number)
+            print(f'Copy {source_path} to {dest_path}')
+            try:
+                os.makedirs(dest_path, exist_ok=True)
+                shutil.copy2(source_path, dest_path)
+            except Exception as e:
+                print('COPY FAIL')
+                print(e)
+                exit(1)
+            else:
+                print('COPY OK')
+
+            print(f'Unzip {dest_path}')
+            extended_path = "\\\\?\\%s" % os.path.join(dest_path, LOGS_ZIP_NAME)
+            with zipfile.ZipFile(extended_path, "r") as zipObj:
+                for name in zipObj.namelist():
+                    if re.match(f'test/apiconformancetests_({"|".join(HW_DEVICES).lower()})_(dlb|omz)-\d.log', name):
+                        print('File found')
+                        zipObj.extract(name, dest_path)
+                        print('File extracted')
+
+    return local_log_dir
+
+def collect_tests_result(local_log_dir):
+    logs_info = {}
+    analyzed_test = None
+    analyzed_number = None
+    in_run_stage = False
+    error_msg = ''
+    device = ''
+    analyzed_devices = set()
+    for root, dirs, files in os.walk(local_log_dir, topdown=False):
+        for file_name in files:
+            file_name_match = re.match(f'apiconformancetests_({"|".join(HW_DEVICES).lower()})_(dlb|omz)-\d.log', file_name)
+            if "test" in root and file_name_match:
+                extended_path = "\\\\?\\%s" % os.path.join(root, file_name)
+                try:
+                    lines = ''
+                    with open(extended_path, encoding="utf8") as f:
+                        lines = f.readlines()
+
+                    # define hw device by filename
+                    device = file_name_match.group(1)
+                    analyzed_devices.add(device)
+
+                    for line in lines:
+                        if '[ RUN      ]' in line:
```

Comment:
I've added the same constant in https://github.com/openvinotoolkit/openvino/pull/14479. Looks like we can move these condtant to utils